Allow loading bitmap fonts

I want to load bdf or pcx fonts.

Trying to add more types beyond "ttf" got me this error:
```
src/fileio/resmgr/res_font.hpp:30:29: error: redefinition of ‘type* ResMgr::resLoader<type>::operator()(boost::filesystem::path) [with type = sf::Font]’
  template<> inline BDFRsrc* resLoader<FontRsrc>::operator() (fs::path fpath) {
```
Files changed (.cpp and .hpp cannot be uploaded, so they are 'txt')

src/fileio/fileio.cpp
[fileio.txt](https://github.com/calref/cboe/files/4074016/fileio.txt)

src/fileio/resmgr/restypes.cpp
[restypes.txt](https://github.com/calref/cboe/files/4074005/restypes.txt)

src/fileio/resmgr/res_font.hpp
[res_font.txt](https://github.com/calref/cboe/files/4074050/res_font.txt)

Since we're using SFML the minimum code would be:
```
sf::Font plain_font;
plain_font.loadFromFile("./rsrc/fonts/ClortSans-11.bdf"))
```
Can someone please modify the three files linked here to allow other font types besides ttf to be loaded? That would be great.

--- a/rsrc/SConscript
+++ b/rsrc/SConscript
@@ -11,7 +11,7 @@ Import("env platform data_dir install_dir")
 
 env.Install(path.join(data_dir, "cursors"), Glob("cursors/*.gif"))
 env.Install(path.join(data_dir, "dialogs"), Glob("dialogs/*.xml"))
-env.Install(path.join(data_dir, "fonts"), Glob("fonts/*.ttf"))
+env.Install(path.join(data_dir, "fonts"), Glob("fonts/*.ttf") + Glob("fonts/*.bdf"))
 env.Install(path.join(data_dir, "graphics"), Glob("graphics/*.png"))
 env.Install(path.join(data_dir, "sounds"), Glob("sounds/*.wav"))
 env.Install(path.join(data_dir, "strings"), Glob("strings/*.txt"))

